### PR TITLE
Fixed small mistake to make run

### PR DESCRIPTION
**Pull request by JaysonHahn**
## Overview

Fixed small mistake to make run

## Changes Made

Corrected the string to use when converting the range to NSRange

## Test Coverage

Manual testing

## Screenshots (delete if not applicable)

<details>

  <summary>Screen Shot Name</summary>
App does not crash
Simulator Screenshot - iPhone 15 Pro - 2024-02-21 at 13 02 52

  
  

</details>

### CODE REVIEW

**Review by vinnie4k — APPROVED** [review]
Good to go. Next time, it would be helpful to document why this change was necessary and how you came to this conclusion. Let's wait for Caitlyn to review before merging.

**Review by caitlynjin — APPROVED** [review]
Looks good to me!

**Pull request merged**
